xnat-apis/getProjects.js

```diff
@@ -0,0 +1,16 @@
+var fetch = require('node-fetch')
+var { Headers } = require('node-fetch')
+
+const getProjects = async (JSESSIONID) => {
+  const res = await fetch(`${process.env.BASE_XNAT_URL}/data/projects/`
+    , {
```

Comment:
Can this be on the previous line? Or did `standard` fix that for you?

Reply:
oh I guess I did that! hmm seems weird!